christina-ch264/week-6/react-giphy-lab-hw/src/Components/Main/SearchContainer.js

```diff
@@ -10,43 +10,38 @@ class SearchContainer extends Component {
         resultList: []
     }
 
-    // handleInput = (e) => {
-    //     e.preventDefault();
-    //     console.log("Submit")
-    // }
-
 // Pseudocode
 // 1. Define elements in render() using values from state.
 // 2. Capture changes of a form element using onChange() as they happen.
 // 3. Update the internal state in event handler.
 // 4. New values are saved in state and then the view is updated by a new render().
 
-
-
     giphySearch = () => {
-        console.log('axios api goes here')
+        console.log('axios api looks for:', this.state.query);
         // use axios api
-        axios.get('https://api.giphy.com/v1/gifs/search?', {
-            params: {
-                api_key:"Gf2zWMUcvwe2Ot0FzFTFeG2BddE9xTrF",
-                q: this.state.query
-            }
-        })
+        axios.get(`http://api.giphy.com/v1/gifs/search?q=${this.state.query}&api_key=RHi6JOuPn9wR4pGJziqoZ3WE27wzW0pz`)
         .then((response) => {
-            console.log(response.data)
+            console.log(response.data.data);
```

Comment:
Love seeing all the console logs. Doing this is a great way to visualize and see the data you are receiving from how to manipulate it.